src/include/kdbio/ev.h

```diff
@@ -8,7 +8,7 @@
 #ifndef KDB_IOWRAPPER_EV_H_
 #define KDB_IOWRAPPER_EV_H_
 
-#include "kdbio.h"
+#include "../kdbio.h"
```

Comment:
Better to manipulate the include path and include kdbio/kdbio.h here.
I would even keep the kdbio.h top-level so that `<kdbio.h>` can be included.